numalogic/connectors/druid/postaggregator.py

```diff
@@ -0,0 +1,19 @@
+from pydruid.utils.postaggregator import Field
+from pydruid.utils.postaggregator import Postaggregator
+
+
+class QuantilesDoublesSketchToQuantile(Postaggregator):
+    """Class for building QuantilesDoublesSketchToQuantile post aggregator."""
+
+    def __init__(self, field: Field, fraction: float, output_name=None):
+        name = output_name or "quantilesDoublesSketchToQuantile"
```

Comment:
Do we need this name variable for the query?

cc @ab93

Reply:
Let's keep it for now. No harm